system_tests/fuzzing/dump_contents.test.disabled

```diff
@@ -1,10 +1,12 @@
 # This test fuzzes input to the dump utility's --contents switch.
 
-# %binaries = the directories containing the executable binaries
-# %stores = a directory in which data stores may be created
-# %T = the test output directory
 # %S = the test source directory
+# %binaries = the directories containing the executable binaries
+# %t = temporary file name unique to the test
```

Comment:
%t = temporary file name? should change it to the temporary folder?
It looks like %t is used as either temporary file name or the temporary folder. Should we give a different definition here?

Reply:
This definition is copied directly from the [LIT documentation](https://llvm.org/docs/CommandGuide/lit.html#substitutions). You’re right that in some of the tests this name is used to create a directory when the test uses more than one intermediate file.